### ISSUE TITLE
Cluster registration fields are not being set when using Rancher master-head (Rancher 2.6)

### ISSUE THREAD

**Issue by git-ival** [rancher/terraform-provider-rancher2#703]
I’ve run into an issue during preliminary scale-testing against Rancher master-head using the terraform scaling framework. The issue is that the rancher2 provider isn't able to retrieve many of the `cluster_registration_token` fields at the time of provisioning via the scaling framework, but seem to be available when viewed in the Rancher API. Whereas using Rancher 2.5.8 populates all the expected fields at the time of provisioning. This causes clusters provisioned via the rancher2 terraform provider to not be able to register with Rancher on master-head (8d03e07).

To clarify, the API itself contains the values needed, but it seems that the rancher2 provider is not able to grab them from Rancher on master-head (8d03e07). 

Versions:
Terraform: 1.0.0
Rancher2 Provider 1.16.0
Scaling Framework: https://github.com/rancher/rancher2-scaling/commit/1658004e9c9cd81182fa324490ab588580cb5be7

Below is an example of the difference between the terraform provider output for master-head and 2.5.8:
```
(Rancher master-head 8d03e07) rancher2_cluster.k3s[*].cluster_registration_token[0] =
 {
  "annotations" = tomap({})
  "cluster_id" = "c-sgvjj"
  "command" = ""
  "id" = "c-sgvjj:system"
  "insecure_command" = ""
  "labels" = tomap({
   "cattle.io/creator" = "norman"
  })
  "manifest_url" = ""
  "name" = "system"
  "node_command" = ""
  "token" = "<token>"
  "windows_node_command" = ""
 }

(Rancher 2.5.8) rancher2_cluster.k3s[*].cluster_registration_token[0] =
{
  "annotations" = tomap({})
  "cluster_id" = "c-2x2m2"
  "command" = "kubectl apply -f <rancher_url>/v3/import/k4q2kzr98vtjbcr6pgpk5mnsg59hvbbtg9bfg5kj5rh7d9nkddmbwj_c-2x2m2.yaml"
  "id" = "c-2x2m2:system"
  "insecure_command" = "curl --insecure -sfL <rancher_url>/v3/import/k4q2kzr98vtjbcr6pgpk5mnsg59hvbbtg9bfg5kj5rh7d9nkddmbwj_c-2x2m2.yaml | kubectl apply -f -"
  "labels" = tomap({
   "cattle.io/creator" = "norman"
  })
  "manifest_url" = "<rancher_url>/v3/import/k4q2kzr98vtjbcr6pgpk5mnsg59hvbbtg9bfg5kj5rh7d9nkddmbwj_c-2x2m2.yaml"
  "name" = "system"
  "node_command" = "sudo docker run -d --privileged --restart=unless-stopped --net=host -v /etc/kubernetes:/etc/kubernetes -v /var/run:/var/run rancher/rancher-agent:v2.5.8 --server <rancher_url> --token "<token>"
  "token" = "<token>"
  "windows_node_command" = "PowerShell -NoLogo -NonInteractive -Command \"& {docker run -v c:\\:c:\\host rancher/rancher-agent:v2.5.8 bootstrap --server <rancher_url> --token <token> | iex}\""
 }
 ```

**Comment by rawmind0** [comment]
Rancher master-head is not supported by this provider, just Rancher stable versions. Anyway, found the issue cause ad submitted PR https://github.com/rancher/terraform-provider-rancher2/pull/709 to fix it.

**Comment by rawmind0** [comment]
PR #709 already merged. The fix will be included in next tfp release, v1.16.1
